### PR TITLE
Replace `<stuff>' with '<stuff>'

### PR DESCRIPTION
**Pull request by owen-mc-diffblue**
This changes the error message from
failed to load class `java.nio.file.Path'
to
failed to load class 'java.nio.file.Path'

- [ ] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Comment by danpoe** [comment]
That's probably intentional as the pattern with left hand ``` ` ``` and right hand `'` occurs more frequently in the code than two single quotes.

**Comment by owen-mc-diffblue** [comment]
Thanks for pointing that out @danpoe , on closer inspection I see that you are right. In that case, I'm expanding the scope of this PR to replace ` with ' everywhere where the closing delimiter is '. I think this is a much more common usage of punctuation.

**Comment by tautschnig** [comment]
@owen-jones-diffblue Why is this a desirable change?

**Comment by kroening** [comment]
This is certainly a very unimportant point one could get very excited about.
https://www.cl.cam.ac.uk/~mgk25/ucs/quotes.html
We could add `quotebegin` and `quoteend` stream modifiers. The result could then be quotes, a change in font, etc., depending on target.

**Comment by owen-mc-diffblue** [comment]
@tautschnig As the link that @kroening posted says, the convention of quoting `text like this' looks odd in most modern fonts.

I appreciate this is not a majorly important change, but it's something that annoys me. I think it makes the output look unprofessional. It was a treat I gave myself after doing some very tedious work I didn't really want to do. The prompt was someone asking about an error message, and they obviously thought it was a mistake that there was a ` instead of a '.

**Comment by tautschnig** [comment]
@owen-jones-diffblue Thank you for elaborating - it would be really helpful to have the "Why?" answered in commit messages (always, not just here). That settled, we are probably very inconsistent about our quoting and thus this is a really good step forward! My only reservation is that there may be users parsing warnings and thus we'd (negatively) surprise them, but arguably no one should rely on formatting of plain-text messages so I'm ok to ignore this one.

File: jbmc/regression/jbmc/generics_type_param/test.desc [hunk 1]
```diff
@@ -7,6 +7,6 @@ Reading class AWrapper
 Reading class FWrapper
 Reading class IWrapper
 --
-failed to load class \`AWrapper\'
-failed to load class \`FWrapper\'
-failed to load class \`IWrapper\'
+failed to load class \'AWrapper\'
+failed to load class \'FWrapper\'
+failed to load class \'IWrapper\'
```
Comment (tautschnig) [inline jbmc/regression/jbmc/generics_type_param/test.desc]: Nit picking: why are we using escapes here?

**Review by tautschnig — APPROVED** [review]

**Review by peterschrammel — COMMENTED** [review]
I actually like @kroening's suggestion of stream modifiers... For the next rainy day...

File: jbmc/regression/jbmc/generics_type_param/test.desc (hunk 1 above)
Comment (owen-mc-diffblue) [reply jbmc/regression/jbmc/generics_type_param/test.desc]: I agree, and it's probably closely enough related to this PR to include it

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: 8e86fe201a0277ffcff2837248b2fe0b68f09838).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/118429422

**Review by smowton — APPROVED** [review]

**Review by chrisr-diffblue — APPROVED** [review]
Looks good to me. I'd also +1 on the idea of having stream modifiers. That would be very helpful for the use-case of wanting to be able to 'redact' the output of tools, such as in cases where we would like to have the end-user be able to send us output from the tools, but where they do not want to leak any information about their own input source code. With stream modifiers it would be simple to add a redaction mode or other tokens that could be used for post-processing redaction.

**Review by romainbrenguier — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: 4c900d1bf8dda708f9b63812114faecf366490f5).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/121070770

**Comment by owen-mc-diffblue** [comment]
TG bump passed CI

**Pull request merged**